### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by wonwon0**
readme created

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# ssl-refbox
+RoboCup Small Size League Referee Box
+
+
+to install:
```
Comment (g3force) [inline README.md]: Make this a title (for example ## Installation or ## Build)

File: README.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+1: clone the repro somewhere
+
+2: enter the directory of the git repro and open a terminal there.
+
+3: in a terminal type: ./installDeps.sh
+
+4: create a build driectory: To do so type "mkdir build" in the terminal
```
Comment (g3force) [inline README.md]: This and the following steps are already contained in the `Makefile`. Simply running `make` will automatically create the build directory, run cmake and make in the build folder.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+5: navigate in the build directory: "cd build"
+
+6: generate the files: "cmake .."
+
+7: compile the project: "make"
+
+8: the run the refbox, go back in the repro root and run ./ssl-refbox in a terminal
```
Comment (g3force) [inline README.md]: to run

File: README.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
+# ssl-refbox
+RoboCup Small Size League Referee Box
+
+
+to install:
+
+1: clone the repro somewhere
```
Comment (g3force) [inline README.md]: isn't this quite obvious? ;)

Comment (wonwon0) [reply README.md]: this is not a complaint nor a judgment, some people that are, for example,  not quite familiar with coding might want to install ssl-refbox to play with it. I made these steps to be fool-proof for someone who has almost 0 experience with coding.

File: README.md (hunk 2 above)
Comment (g3force) [reply README.md]: Simply run `make` to build the project. No need to deal with cmake, that's done by the Makefile. Try it out ;)

**Comment by g3force** [comment]
I've noticed that detailed installation instructions are documented as github pages. I created a README file that points to those pages and close this one.

**Pull request closed without merging**